src/app/shared/services/local-storage.ts

```diff
@@ -0,0 +1,17 @@
+export const LocalStorage = {
```

Comment:
move this to LocalStorageService or to AuthService, constants for static data from my point of view